src/ixdat/techniques/ec_ms.py

```diff
@@ -426,6 +429,156 @@ def siq_ecms_calibration_curve(
         else:
             return cal
 
+    # DECONVOLUTION METHODS BELOW
+    def grab_deconvoluted_current(
+        self, mol, impulse_response, tspan=None, tspan_bg=None, snr=10
+    ):
+        """Return the deconvoluted partial current for a given signal using the
+        algorithm developed by Krempl et al.
+        https://pubs.acs.org/doi/abs/10.1021/acs.analchem.1c00110
+
+        Note, this actually doesnt need the EC data - it is calculated
+        from calibrated MS data. It is only meaningful for ECMS measurements
+        at the moment, justifying placement here.
+
+        Args:
+            mol (str): Name of molecule for which deconvolution is to
+                be carried out.
+            impulse_response (ECMSImpulseResponse): Kernel object which contains the mass
+                transport parameters
+            tspan (list): Timespan for which the partial current is returned.
+            tspan_bg (list): Timespan that corresponds to the background signal.
+            snr (int): signal-to-noise ratio used for Wiener deconvolution.
```

Comment:
What does it return exactly? What units? 
It's called `partial_current` but I can't see an `n_el` (number of electrons per molecule) anywhere, so I'm not sure how it could actually be so.
Looks from line 478 like it must be the same as `v_sig_corr`, which is [mol/s]?

Reply:
yeah exactly

Reply:
Can you add a description of the return at the end of the docstring?